### PR TITLE
MAISTRA-1374: support multiple multus CNI interfaces

### PR DESCRIPTION
**Pull request by knrc**
This needs to be updated once https://github.com/Maistra/istio/pull/117 is merged

### CODE REVIEW

File: build/patch-charts.sh [hunk 1]
```diff
@@ -254,12 +254,6 @@ function patchSidecarInjector() {
       - SETUID
   }' ${HELM_DIR}/istio/files/injection-template.yaml
 
-  # add annotation for Multus & Istio CNI
-  sed_wrap -i -e 's/^\(.*injectedAnnotations:.*\)$/\1\
-    \{\{- if .Values.istio_cni.enabled \}\}\
-      k8s.v1.cni.cncf.io\/networks: \{\{.Values.istio_cni.istio_cni_network\}\}\
-    \{\{- end \}\}/' ${HELM_DIR}/istio/templates/sidecar-injector-configmap.yaml
```
Comment (dgn) [inline build/patch-charts.sh:261]: I'm afraid we still need this, as the injection of `podRedirectAnnot` is disabled by default in 1.1. We should replace the check for `.Values.istio_cni.enabled` (as that'll always be true with Maistra) with a check for `not .Values.sidecarInjectorWebhook.injectPodRedirectAnnot`

cc @luksa

**Review by dgn — CHANGES_REQUESTED** [review]

File: build/patch-charts.sh (hunk 1 above)
Comment (knrc) [reply build/patch-charts.sh:261]: I thought we had recently enabled this, I'll double check today and fix

Comment (knrc) [reply build/patch-charts.sh:261]: I've verified it's not enabled by default, apologies for this.  I'll rework the fix.

Comment (knrc) [reply build/patch-charts.sh:261]: @dgn Can you please take another look?

**Review by dgn — APPROVED** [review]

File: build/patch-charts.sh (hunk 1 above)
Comment (dgn) [reply build/patch-charts.sh:261]: LGTM, thanks

**Review by rcernich — APPROVED** [review]

**Pull request merged**